rhino/src/main/java/org/mozilla/javascript/ScriptableObject.java

```diff
@@ -1690,6 +1691,64 @@ public void defineProperty(
         slot.setter = setter;
     }
 
+    /**
+     * Define a property on this object that is implemented using lambda functions accepting
+     * Scriptable `this` object as first parameter. Unlike with `defineProperty(String name,
+     * Supplier<Object> getter, Consumer<Object> setter, int attributes)` where getter and setter
+     * need to have access to target object instance, this allows for defining properties on
+     * LambdaConstructor prototype providing getter and setter logic with java instance methods. If
+     * a property with the same name already exists, then it will be replaced. This property will
+     * appear to the JavaScript user exactly like any other property -- unlike Function properties
```

Comment:
I'm sorry I keep finding things, but now this comment is out of date-- as you've defined it, and unlike the other "defineProperty" that takes  Getter and a Setter, this property will not look like a normal value, but like a descriptor with a getter and setter, just as if they had been defined in JavaScript using Object.defineOwnProperty. Could you please update? I think this will be the last thing!

Reply:
Of course, it's not great to leave outdated comments! Especially in this case where it can already be pretty confusing which `defineProperty` should be used. I should have noticed this myself, thanks for being patient with me!